frontend/package.json

```diff
@@ -119,6 +119,7 @@
     "suppress-exit-code": "^1.0.0",
     "terser-webpack-plugin": "^5.3.6",
     "toml": "^3.0.0",
+    "vercel": "^28.16.15",
```

Comment:
Should this not be in devDependencies?

Reply:
No, because our actions do npm ci

Reply:
It's the same reason why webpack etc is there too.